synapse/replication/http/_base.py

```diff
@@ -182,7 +189,7 @@ def make_client(cls, hs: "HomeServer"):
             )
 
         @trace(opname="outgoing_replication_request")
-        async def send_request(*, instance_name="master", **kwargs):
+        async def send_request(*, instance_name: str = "master", **kwargs: Any) -> Any:
```

Comment:
Again this is very weak, but I think they can return anything? This matches the definition of `request_func` below.

Reply:
Yeah this is the whole "how do you represent a JSON result" issue.